src/stub/parser.rs

```diff
@@ -178,7 +178,7 @@ impl<'a> Parser<'a> {
     // Doesn't deal with InputComments to unassigned variables
     // nor InputComments to variables with the same identifier
     fn parse_input_comment(&mut self, previous_commands: &mut [Cmd]) {
-        self.skip_to_next_line();
+        self.seek_to_next_line();
```

Comment:
I find `seek_to` ambiguous, compared to `skip`. 

If you really didn't want `skip_to_next_line` I guess you could use `discard_line`, `skip_line`, `consume_line`, `move_to_next_line` etc.

Reply:
Fair enough, I was just thinking of using the typical lingo when you seek through a file. I'll switch it back.

Reply:
Maybe it's my bad.

Do you have any references that I can check? I don't think I have seen `seek` with the meaning of consuming / discarding myself.

Reply:
And honestly since you are at it, I don't know what you think but for some reason I find `skip_line` clearer than `skip_to_next_line`.

Reply:
> Do you have any references that I can check? I don't think I have seen `seek` with the meaning of consuming / discarding myself.

https://ruby-doc.org/core-2.5.1/IO.html#method-i-seek

It's not the context of discarding but moving a pointer (in our case thru the iterator).


> And honestly since you are at it, I don't know what you think but for some reason I find `skip_line` clearer than `skip_to_next_line`.

That's probably the best option, keeping it simple.